.github/workflows/test_cra.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, review_requested]
+  issue_comment:
+    types: [created]
+
+jobs:
+  code_review_job:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run code review agent on every pull request, respond to user comments
+
+    steps:
+      - name: Set Options Environment Variable for Issue Comment
+        if: github.event_name == 'issue_comment'
+        run: |
+          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${{ secrets.GIT_DOMAIN && ' --git.domain=' || '' }}${{ secrets.GIT_DOMAIN }}" >> $GITHUB_ENV
+      - name: Code Review Agent - Issue Comment
+        if: github.event_name == 'issue_comment'
+        uses: gitbito/codereviewagent@BITO_QA_TEST
+        with:
+          pr: ${{ github.event.issue.pull_request.html_url }}
+          command: ${{ github.event.comment.body }}
+          options: ${{ env.OPTIONS }}
+
+      - name: Set Options Environment Variable for Pull Request
+        if: github.event_name == 'pull_request'
+        run: |
+          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${{ secrets.GIT_DOMAIN && ' --git.domain=' || '' }}${{ secrets.GIT_DOMAIN }}" >> $GITHUB_ENV
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider consolidating duplicated options string</b></div>
<div id="fix">

Consider extracting the common options string into a reusable environment variable at the workflow level to avoid duplication. This could help with maintainability and reduce the risk of inconsistencies.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```
 @@ -1,39 +1,42 @@
  on:
    pull_request:
      types: [opened, reopened, ready_for_review, review_requested]
    issue_comment:
      types: [created]

 +env:
 +  BASE_OPTIONS: --static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${{ secrets.GIT_DOMAIN && ' --git.domain=' || '' }}${{ secrets.GIT_DOMAIN }}
 +
  jobs:
    code_review_job:
      runs-on: ubuntu-latest
      permissions:
        issues: write
        pull-requests: write
        contents: write
      name: Run code review agent on every pull request, respond to user comments

      steps:
        - name: Set Options Environment Variable for Issue Comment
          if: github.event_name == 'issue_comment'
          run: |
 -          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${{ secrets.GIT_DOMAIN && ' --git.domain=' || '' }}${{ secrets.GIT_DOMAIN }}" >> $GITHUB_ENV
 +          echo "OPTIONS=${{ env.BASE_OPTIONS }}" >> $GITHUB_ENV

        - name: Set Options Environment Variable for Pull Request
          if: github.event_name == 'pull_request'
          run: |
 -          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${{ secrets.GIT_DOMAIN && ' --git.domain=' || '' }}${{ secrets.GIT_DOMAIN }}" >> $GITHUB_ENV
 +          echo "OPTIONS=${{ env.BASE_OPTIONS }}" >> $GITHUB_ENV
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Amruta101998/DjangoProject/pull/8#issuecomment-2544863304>#985c01</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged